discrete_optimization/vrp/solver/lp_vrp_iterative_pymip.py

```diff
@@ -406,82 +406,81 @@ def solve(self, **kwargs):
                 best_solution_objective_rebuilt = obj
                 best_solution_rebuilt_index = iteration
             iteration += 1
-            if len(component_global_all[0]) > 1 or True:
```

Comment:
same comment as the lp_solver_pymip (which is very similar). for this one, better to introduce an argument to the function

Reply:
Same question: by default do we want to bypass the block code?

Reply:
Ok my comment was a bit wrong, re-reading the full code, the correct fix is to keep the "if len(component_global_all[0]) > 1" condition, and remove the True

Reply:
Taken into account